src/main/scala/encry/api/http/routes/NodeApiRoute.scala

```diff
@@ -0,0 +1,25 @@
+package encry.api.http.routes
+
+import akka.actor.{ActorRef, ActorRefFactory}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import encry.EncryApp
+import encry.settings.RESTApiSettings
+
+case class NodeApiRoute(restApiSettings: RESTApiSettings)(implicit val context: ActorRefFactory)
```

Comment:
I'm not sure, that creating such a route is a good idea

Reply:
This is necessary for the correct node restart with other port
So as not to damage the database files

We can refuse the route and take a chance